.github/workflows/ccpp.yml

```diff
@@ -42,7 +42,7 @@ jobs:
                   g++-multilib \
                   dejagnu;
           # install Rust directly using rustup
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=nightly;
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=1.72.0;
```

Comment:
Out of curiosity, any reason for using 1.72 here? Latest is 1.80, so that's several  release behind.

Reply:
not most stable, most recent*